src/app/component/add-product/add-product.component.html

```diff
@@ -0,0 +1,69 @@
+<section class="hero is-fullheight">
+    <div class="hero-body">
+        <div class="container">
+            <div class="level">
+                <div class="level-left">
+                    <div class="level-item">
+                        <p class="title is-2">
+                            New Product
+                        </p>
+                    </div>
+                </div>
+            </div>
+
+            <form [formGroup]="productForm" (ngSubmit)="onSubmit()">
```

Comment:
Have a look over this component separation technique: https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0